app/cdap/components/GridTable/index.tsx

```diff
@@ -35,7 +36,7 @@ import { useStyles } from './styles';
 import { IExecuteAPIResponse, IHeaderNamesList, IObject, IParams, IRecords } from './types';
 import { convertNonNullPercent } from './utils';
 
-export default function GridTable() {
+export default function() {
```

Comment:
why is this change done here, this isn't need for footer panel to work
plz revert unwanted changes